### PR TITLE
Clarify a few details around Neo4j plugins

### PR DESCRIPTION
**Pull request by NataliaIvakina**
Add a footnote that GenAI plugin is available in EE only
Remove mentioning about the Ops Manager, as it doesn't run as a plugin but as a separate product
Mention the Configure plugins page in the intro page to the section
Clarify that Bloom Enterprise and GDS Enterprise require license key

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1636.surge.sh

Updated pages:
- [Configuration](https://neo4j-docs-operations-1636.surge.sh/operations-manual/5/configuration)
- [Configure plugins](https://neo4j-docs-operations-1636.surge.sh/operations-manual/5/configuration/plugins)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1636.surge.sh

Updated pages:
- [Configuration](https://neo4j-docs-operations-1636.surge.sh/operations-manual/5/configuration)
- [Configure plugins](https://neo4j-docs-operations-1636.surge.sh/operations-manual/5/configuration/plugins)

File: modules/ROOT/pages/configuration/plugins.adoc [hunk 1]
```diff
@@ -19,46 +21,51 @@ The following plugins are supported:
...
 | https://neo4j.com/docs/apoc/current/[APOC user guide]
 
-| Bloom
+| Bloom Enterprise footnote:[You can also get basic access to Bloom via Graph Apps in Neo4j Desktop without a license key. See link:{neo4j-docs-base-uri}/bloom-user-guide/current/bloom-installation/bloom-deployment-modes/[Bloom deployment modes] for more details.]
 | `bloom`
 | {check-mark}
 | link:{neo4j-docs-base-uri}/bloom-user-guide[Neo4j Bloom]
 
-| GenAI
+| GenAI footnote:[The GenAI plugin is only available in Neo4j **Enterprise** Edition from 5.17.]
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/configuration/plugins.adoc:41]:
```suggestion
| GenAI footnote:[The GenAI plugin is only available in Neo4j **Enterprise** Edition starting from Neo4j 5.17.]
```

File: modules/ROOT/pages/configuration/plugins.adoc [hunk 2]
```diff
@@ -2,15 +2,17 @@
 = Configure plugins
 :description: This page describes how to load plugins to a Neo4j deployment.
 
-Neo4j distributions come bundled with a range of pre-installed products, such as Graph Data Science library, Bloom, and Ops Manager, which can be used to extend the Neo4j capabilities.
-The JAR files for these products are located in the _product_ and _labs_ folders and can be installed as plugins.
+Neo4j Enterprise Edition comes bundled with a range of pre-installed products, such as Graph Data Science library or Bloom, which can be used to extend the Neo4j capabilities.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/configuration/plugins.adoc:5]:
```suggestion
Neo4j Enterprise Edition comes bundled with a range of pre-installed products, such as Graph Data Science library and Bloom, which can be used to extend the Neo4j capabilities.
```

File: modules/ROOT/pages/configuration/plugins.adoc [hunk 3]
```diff
@@ -2,15 +2,17 @@
...
+
+Neo4j Community Edition comes with the APOC plugin only in the _labs_ directory.
 
 If you want to use your own plugins, ensure that you add them to the designated _plugins_ directory.
 This directory serves as the central location where Neo4j looks for and loads the plugins at startup.
 
 [NOTE]
 ====
-Bloom and GDS Enterprise require a license activation key to become available to the user within Neo4j.
+Bloom Enterprise and GDS Enterprise require a license activation key to become available to the user within Neo4j.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/configuration/plugins.adoc:15]:
```suggestion
Bloom Enterprise and GDS Enterprise require a license activation key.
```

File: modules/ROOT/pages/configuration/plugins.adoc [hunk 4]
```diff
@@ -74,7 +81,7 @@ For more information on using plugins in a different Neo4j setup, see:
 Here are the steps to enable the plugins:
 
 . Move or copy the plugins (_.jar files_) from _<NEO4J_HOME>/products_ and _<NEO4J_HOME>/labs_ to the _<NEO4J_HOME>/plugins_ directory.
-See the table in xref:configuration/file-locations.adoc[File locations] for more information.
+See tables in xref:configuration/file-locations.adoc[Default file locations] for more information.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/configuration/plugins.adoc:84]:
```suggestion
See xref:configuration/file-locations.adoc[Default file locations] for more information.
```

**Review by AlexicaWright — COMMENTED** [review]
Looking good, just a few editorial suggestions. Thank you @NataliaIvakina !

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1636.surge.sh

Updated pages:
- [Configuration](https://neo4j-docs-operations-1636.surge.sh/operations-manual/5/configuration)
- [Configure plugins](https://neo4j-docs-operations-1636.surge.sh/operations-manual/5/configuration/plugins)

**Comment by NataliaIvakina** [comment]
@AlexicaWright, @nvitucci, as I understand, the PR is ready to be merged, isn't it?

**Review by AlexicaWright — APPROVED** [review]

**Pull request merged**